server/server/src/schema.rs

```diff
@@ -49,9 +47,34 @@ pub mod v2 {
             app_store_ids: <String, Owner>
         }
     }
+}
+
+pub mod v3 {
+    use super::Account;
+    use lockbook_shared::file_metadata::Owner;
+    use lockbook_shared::server_file::ServerFile;
+    use std::collections::HashSet;
+    use uuid::Uuid;
+
+    hmdb::schema! {
+        Server {
+            usernames: <String, Owner>,
+            accounts: <Owner, Account>,
+            owned_files: <Owner, HashSet<Uuid>>,
+            shared_files: <Owner, HashSet<Uuid>>,
+            metas: <Uuid, ServerFile>,
+            file_children: <Uuid, HashSet<Uuid>>,
+            sizes: <Uuid, u64>,
+            google_play_ids: <String, Owner>,
+            stripe_ids: <String, Owner>,
+            app_store_ids: <String, Owner>,
+            last_seen: <Owner, u64>
+        }
+    }
 
     pub fn migrate(
-        source: &mut super::transaction::ServerV1, destination: &mut transaction::Server,
```

Comment:
can you order the operations in the order that they appear in v3? makes it easier to make sure you didn't miss one, something tests or compiler will not check for